Gruntfile.coffee

```diff
@@ -41,6 +48,12 @@ module.exports = (grunt) ->
     shell:
       webdriverManagerUpdate:
         command: "node_modules/grunt-protractor-runner/node_modules/.bin/webdriver-manager update"
+      generateApiDocs:
+        command: [
+          "node_modules/docha/bin/docha -p '.tmp/elementHelper.spec.js' -o 'docs/elementHelper.md' -e _"
+          "node_modules/docha/bin/docha -p '.tmp/generalStepDefs.spec.js' -o 'docs/generalStepDefs.md' -e _"
+          "node_modules/docha/bin/docha -p '.tmp/transform.spec.js' -o 'docs/transform.md' -e _"
```

Comment:
Thanks for the reminder! In fact, as @dansomething mentioned in a [comment on a previous pull request](https://github.com/ReadyTalk/cukefarm/pull/5#discussion_r26864126) these can even be shortened further to just `docha`.